tune-threads/tune-threads.lisp

```diff
@@ -0,0 +1,91 @@
+;;;; tune-threads/tune-threads.lisp
+;;;;
+;;;; Author: John Lapeyre
+
+(in-package #:qvm-tune-threads)
+
+(defun reset-qvm (q)
```

Comment:
I would be supportive of having a method `reset-qvm-state` or something like that in the core QVM code. Could you add it there? I recommend adding it to `qam.lisp` and `qvm.lisp`